src/components/contextual/pages/pool/PoolMigrationCard/PoolMigrationCard.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+import useNetwork from '@/composables/useNetwork';
+import { deprecatedDetails } from '@/composables/usePool';
+import NewPoolData from './NewPoolData.vue';
+import usePoolsQuery from '@/composables/queries/usePoolsQuery';
+import usePoolQuery from '@/composables/queries/usePoolQuery';
+import GradientCard from '@/components/cards/GradientCard/GradientCard.vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  poolId: string;
+};
+
+/**
+ * PROPS
+ */
+const props = defineProps<Props>();
+
+/**
+ * COMPOSABLES
+ */
+const { networkSlug } = useNetwork();
+
+const poolDeprecatedDetails = computed(() => deprecatedDetails(props.poolId));
+
+let newPoolQuery;
+if (poolDeprecatedDetails.value?.newPool) {
+  newPoolQuery = usePoolQuery(poolDeprecatedDetails.value?.newPool);
+}
```

Comment:
To prevent the queries from running you can just use these conditionals as the enabled flag in the options.
```suggestion
const newPoolQueryEnabled = computed((): boolean => !!poolDeprecatedDetails.value?.newPool);
newPoolQuery = usePoolQuery(poolDeprecatedDetails.value?.newPool, newPoolQueryEnabled);
```
Same with the suggested pool query.

Reply:
yeah, exactly